components/Tabs2/src/private/MDCTabBarViewItemView.m

```diff
@@ -23,8 +23,82 @@
 /** The minimum height of any item view. */
 static const CGFloat kMinHeight = 48;
 
+/// Outer edge padding from spec: https://material.io/go/design-tabs#spec.
+static const UIEdgeInsets kEdgeInsets = {.top = 12, .right = 16, .bottom = 12, .left = 16};
+
+/// File name of the bundle (without the '.bundle' extension) containing resources.
+static NSString *const kResourceBundleName = @"MaterialTabs";
+
+/// String table name containing localized strings.
+static NSString *const kStringTableName = @"MaterialTabs";
+
+/// Scale factor applied to the title of bottom navigation items when selected.
+const CGFloat kSelectedNavigationTitleScaleFactor = (16.0f / 14.0f);
+
+/// Vertical translation applied to image components bottom navigation items when selected.
+const CGFloat kSelectedNavigationImageYOffset = -2;
```

Comment:
This is not part of Tabs - it's part of Bottom Navigation. It also looks unused.  Please remove it if possible.

Reply:
Done.